source/replication.rst

```diff
@@ -195,3 +202,43 @@ The *Replica* site will be ready for replication when you apply changes as usual
 
    $ kubectl apply -f deploy/cr.yaml
 
+.. _operator-replication-mcs:
+
+Enabling multi-cluster Services
+--------------------------------------------
+
+Kubernetes `multi-cluster Services (MCS) <https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-services>`_
+is a cross-cluster discovery and invocation of Services. MCS-enabled Services become discoverable and accessible across clusters with a virtual IP address.
+
+This feature allows splitting applications into multiple clusters, which can be
+useful to separate logically standalone parts (i.e. stateful and stateless
+ones), or to address privacy and scalability requirements, etc.
+
+Multi-cluster Services should be supported by the cloud provider. It is
+supported `by Google Kubernetes Engine (GKE) <https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-services>`__, and `by Amazon Elastic Kubernetes Service (EKS) <https://aws.amazon.com/ru/blogs/opensource/introducing-the-aws-cloud-map-multicluster-service-controller-for-k8s-for-kubernetes-multicluster-service-discovery/>`__.
+
+Configuring your cluster for multi-cluster Services includes two parts:
+
+* configure MSC with your cloud provider,
+* make needed preparations with the Operator.
+
+To set up MCS for a specific cloud provider you should follow official guides,
+for example ones `from Google Kubernetes Engine (GKE) <https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services>`__, or `from Amazon Elastic Kubernetes Service (EKS) <https://aws.amazon.com/ru/blogs/opensource/introducing-the-aws-cloud-map-multicluster-service-controller-for-k8s-for-kubernetes-multicluster-service-discovery/>`__.
+
+Setting up the Operator for MCS involves two options in the ``multiCluster``
+subsection of the ``deploy/cr.yaml`` configuration file:
+
+* ``multiCluster.enabled`` should be set to ``true``,
+* ``multiCluster.DNSSuffix`` string should be a cluster domain suffix for
+  the multi-cluster Services.
```

Comment:
for now we support only clusterset suffix. If you want use custom prefix you need to check how resolve dns in k8s and do some changes from k8s side. 
@spron-in @hors is ok to say this way?

Reply:
Also we might say a couple of words about import/export services and that we need to wait until they arise (it could take a couple minutes 5-10)
+ I believe we need to add that if we use the same namespace name we will export all services from all available clusters with the same namespace.
I plan to duplicate this info in blogpost but I believe it could be helpful in this doc too.

Reply:
We also can add a couple of words about vpc limitations 
https://stackoverflow.com/questions/70323945/failed-to-export-the-service-using-gke-multi-cluster-service
and for aws. Just say they customers need to check it. It could be crucial (cross-site replication there are no such limitations)

Reply:
We also need to document how to enable MCS for a running cluster after upgrading from 1.11.0

`*.clusterset.local` SANs needs to be in the TLS certificates. After upgrading from 1.11.0 or below, users should rotate their certificates by pausing the cluster and deleting the TLS secrets.

@spron-in @hors